actions/auth-application/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "auth-application",
+  "version": "1.0.2",
```

Comment:
Should this be `1.0.0`?

Reply:
[0b8daa1](https://github.com/teleport-actions/root/pull/20/commits/0b8daa1cf5964f5b60b11c629e867860b25b3fb3)